nighthawk/include/nighthawk/common/sequencer.h

```diff
@@ -31,16 +32,14 @@ class Sequencer {
   virtual double completionsPerSecond() const PURE;
 
   /**
-   * @return const Statistic& tracks time spend waiting on SequencerTarget while it returns false.
-   * (In other words, time spend while the Sequencer is idle and not blocked by a rate limiter)
-   */
-  virtual const Statistic& blockedStatistic() const PURE;
-
-  /**
-   * @return const Statistic& tracks latency between calling the SequencerTarget and observing its
-   * callback.
+   * Gets a vector of associated Statistics.
+   *
+   * @return StatisticPtrVector A vector of Statistics.
```

Comment:
Why is vector the natural representation? Why merge the two? Why not a map? I think it's because ultimately this is just one of a `repeated` in the output proto, but not sure if it makes sense to abandon typed structure there if there's only two of these. What's the future plans here?

Reply:
Switched to a map, which I agree makes more sense. 

To generate cli/proto output, the current plan is that `Client` will pull the stats from one or more `Worker`(s). These in turn will pull stats from both associated `Sequencer` and `BenchmarkClient`.  The client will then merge stats into an aggregated view, next to the per-worker views.